pom.xml

```diff
@@ -48,7 +48,7 @@
     <parent>
         <groupId>ddf</groupId>
         <artifactId>ddf-parent</artifactId>
-        <version>1.0.0</version>
+        <version>1.0.1-SNAPSHOT</version>
```

Comment:
I'll need to change this since it seems we don't want to use snapshots of ddf-parent.